rdkPlugins/Gamepad/README.md

```diff
@@ -0,0 +1,25 @@
+# Dobby RDK GPU Memory Plugin
```

Comment:
This whole readme file is incorrect. Make sure that it describes your plugin.

Reply:
Corrected